book/website/collaboration/creating-agendas.md

```diff
@@ -0,0 +1,111 @@
+- **M**inutes: Review the minutes of the previous meeting can remind people of what was discussed and decided at the last meeting. This is particuarlly useful for meetings that are infrequent.
+- **I**nformation: Make any FYI announcements. In some meetings the call for information can be a dedicated agenda item (see next section).
+- **N**ext call: When are you meeting next? Now is a good time to pull out your calendars and pick a time! The end of the meeting folks may be running off to their next appointments so getting this out of the way at the start ensures that everyone is acoomindated.
+
+Part of housekeeping is tending to the emotional wellbeing of the group.
+Providing a space for each person to check in with "How are you showing up?" can help people show up as a complete person, provide an opportunity for human-to-human connections, and set asside outside concerns to focus on the meeting.
+Folks sometimes have a hard time naming emotions and it can be helpful to link in a [feelings wheel](https://feelingswheel.com/) to help folks identify what resonates with them in the moment.
+When starting this practice it's very important to be clear that "Pass" is always an OK response.
+Emotional sharing must be at-choice or it edges into cohersion and abuse.
+When used well this can be a powerful connecting tool for your team.
+
+Finally review and consent to the agenda.
+Make sure everyone knows what is going to be talked about and that the group is in agreement that this covers the needs of the group for now.
+
+This opening typically takes 5 to 15 minutes.
+New groups will often require more time in this beginning phase.
+
+## The middle
+
+This is the part that most people think of when they hear 'agenda'.
+Generally items in this section will fall into three categories if the group is trying to come to a shared decision:
+
+1) Sensemaking - What is the context and background that frames a decision? This is a shared experience. One person in the group might have a perfect understanding of the background and context, but if that is not shared and commonly understood, then as a collective the group has not engaged in sensemaking.
+2) Formulation or brainstorming - What are ways we could address the decision in front of us? This is where we get to be expansive and throw ideas around. The goal of this agenda is not to get it 'right' but to gather all possible ideas about something.
+3) Decision - What is the decision we are making as a group? This seems like an obivious topic but it's very easy for a group to leave a meeting with different understandings of what was decided. If a decision is made, say so, ideally in writing.
+
+This is also a good place to check in on sub-circles or individual to-do items.
+A committee has never done anything in the history of time, and a person has never done anything in issolation.
```

Comment:
```suggestion
A committee has never done anything in the history of time, and a person has never done anything in isolation.
```
Maybe a clarification on my end: what do you mean that a committee has never done anything in the history of time? Is this a theoretical synario?

Reply:
Someone has to implement the strategy that the group comes up with. I often see busy researchers discuss and strategize without ever deciding who is going to write the paper, run the experiment, or code the model. Everyone just expects someone else to step in! By specifically naming who is doing what (defining roles) you both recognize folks for the work they are doing and make sure that the group strategy/decision gets implemented.

Is there a clearer way to say this that I'm missing?